lib/matplotlib/mathtext.py

```diff
@@ -1117,7 +1117,7 @@ def _get_font(self, font):
         cached_font = self.fonts.get(basename)
         if cached_font is None:
             fname = os.path.join(self.basepath, basename + ".afm")
-            with open(fname, 'r') as fd:
+            with open(fname, 'rb') as fd:
```

Comment:
Yes, I ran into this line failing while testing stuff, corrected and commited it by mistake. I'll take it out from this PR.